lib/app-oclif.ts

```diff
@@ -46,8 +46,12 @@ export function run(command: string[], options: AppOptions) {
 			// oclif sometimes exits with ExitError code 0 (not an error)
 			if (error instanceof ExitError && error.oclif.exit === 0) {
 				return;
+			} else {
+				throw error;
 			}
-			handleError(error);
 		},
 	);
+	return Promise.all([trackPromise, runPromise]).catch(
+		require('./errors').handleError,
+	);
```

Comment:
This `Promise.all` arrangement is similar to what is currently done for Capitano commands (`app-capitano.coffee`).